whois-nrtm4/src/main/java/net/ripe/db/nrtm4/publish/NrtmObject.java

```diff
@@ -0,0 +1,5 @@
+package net.ripe.db.nrtm4.publish;
+
+public class NrtmObject {
```

Comment:
empty class?

Reply:
yes, it's removed in 'transform-rpsl-to-nrtm-json'